pkg/cache/task.go

```diff
@@ -494,6 +494,10 @@ func (task *Task) beforeTaskCompleted() {
 
 // releaseAllocation sends the release request for the Allocation to the core.
 func (task *Task) releaseAllocation() {
+	// We need to get the termination type from the task, when it is "" we default to the normal termination.
+	// We should log it correctly instead of "" in the logs.
```

Comment:
I think this comment can be removed, it's very obvious why we're doing this

Reply:
Addressed in latest PR, thanks!